realis_e2e_test.go

```diff
@@ -461,6 +465,37 @@ func TestRealisClient_CreateService(t *testing.T) {
 	// Kill task test task after confirming it came up fine
 	_, err = r.KillJob(job.JobKey())
 	assert.NoError(t, err)
+
+	success, err := monitor.Instances(job.JobKey(), 0, 1, 50)
+	assert.True(t, success)
+
+	// Test case where http connection timeouts out.
+	t.Run("TimeoutError", func(t *testing.T) {
+		timeoutClient, err := realis.NewRealisClient(realis.SchedulerUrl(AuroraURL),
+			realis.BasicAuth("aurora", "secret"),
+			realis.TimeoutMS(20))
+		assert.NoError(t, err)
+
+		defer timeoutClient.Close()
+
+		// Make sure a timedout error was returned
```

Comment:
Maybe you should move/copy this logic to example so people know how to handle timeout properly (copy & paste) when doing update timeout.

Also I didn't see test case/example where an invalid job update setting is given, maybe it's already there thus didn't show up in diff view.

Reply:
Invalid payload + timeout should definitely be tested, I'll add that test today. I'll add to the docs how to handle a timeout.